data/csv.go

```diff
@@ -0,0 +1,87 @@
+package data
+
+import (
+	"encoding/csv"
+	"errors"
+	"go-bootcamp/model"
```

Comment:
Just separate the packages using best practices:

Go core

Project Local 

Third-Party